hls-network-throttler.js

```diff
@@ -0,0 +1,72 @@
+const origXhr = videojs.xhr;
```

Comment:
Might be nice at some point to instead have some way of passing the config in to this file somehow.  

hls-network-throttler(config) return the xhr object to set, or set the xhr object as part of initialization.  Might be easier to use it as part of something larger if that change happened.